web-app/app/tricorder/page.tsx

```diff
@@ -1,97 +1,87 @@
 "use client";
 
-import { useEffect, useState } from "react";
+import { useEffect, useRef, useState } from "react";
 import Counter from "./Counter";
-import { Configure } from "./Configure";
+import { Config, ConfigurationDisplay, Configure } from "./Configure";
 
 function Tricorder(props: any) {
-  const scanForLifeforms = useState<any>();
+  const [config, setConfig] = useState<Config>({});
+  const [isScanningForLifeForms, setIsScanningForLifeforms] =
+    useState<boolean>();
+  const [foundLifeforms, setFoundLifeForms] = useState<number[]>([]);
 
-  var isScanningForLifeForms = scanForLifeforms[0];
-  var setScanForLifeFormsToTrue = () => scanForLifeforms[1](true);
-  var setScanForLifeFormsToFalse = () => scanForLifeforms[1](false);
+  const intervalRef = useRef<NodeJS.Timeout>();
```

Comment:
We're tracking the interval here so that we can stop it when we ask it to turn off.